### DISCUSSION TITLE
How do I put code blocks inside other elements?

### DISCUSSION THREAD

**Discussion by icyrockcom** [pngwn/MDsveX#218] [Q&A] [answered]
Sample:

~~~
<div>
  <div class="old">
```js
function oldFunction() {
  console.log('old function');
}
```
  </div>

  <div class="new">
```js
function newFunction() {
  console.log('new function');
}
```
  </div>
</div>
~~~

Pasting this into the [mdsvex playground](https://mdsvex.com/playground), I get this error:

> Expected } (App.svelte:5:29)

Rationale for this: I want to show two (or more) code blocks side-by-side for easy comparison.

As an aside - what an awesome project you've built, thanks a lot! 👍

**Comment by icyrockcom** [comment, marked as answer]
Ah, leaving a blank line before the code block, i.e.:

~~~
<div>
  <div class="old">

```js
function oldFunction() {
  console.log('old function');
}
```
  </div>

  <div class="new">

```js
function newFunction() {
  console.log('new function');
}
```
  </div>
</div>
~~~

solves the issue.

**Comment by pngwn** [comment]
Yeah this requires new line at the minute. I actually have plans to address this limitation in a future version.
